cluster-up/check.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
could cluster-up be a git submodule?

Reply:
@fabiand I had bad times working with sub-modules. I would suggest to keep it as is for now, and later when we will have CI, we can always bump it the same way as we do in KV.